client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -316,6 +337,7 @@ export const getRowMetaData = (
 const formatStatsData = ( site: Site ) => {
 	const statsData: StatsNode = {
 		type: 'stats',
+		status: 'active',
```

Comment:
Is this required/used anywhere? If not, we can get rid of this.

Reply:
I was getting an error message:

`Property 'status' is missing in type '{ type: "stats"; data: SiteStats; }' but required in type 'StatsNode'`

Which when I removed the `status: AllowedStatusTypes | string;` from the `StatsNode`, that took care of it.

Reply:
Now that I removed that, I'm getting a warning on these lines (299-301)

```
	const { link, isExternalLink } = getLinks( type, row.status, siteUrl, siteUrlWithScheme );
	const tooltip = getTooltip( type, row.status );
	const eventName = getRowEventName( type, row.status, isLargeScreen );
```

It's saying `Property 'status' does not exist on type 'StatsNode'.`

Reply:
Looks like 

```
export interface RowMetaData {
	row: {
		value: Site | any;
		status: AllowedStatusTypes | string;
		error?: boolean;
	};
```

in /sites-overview/types.ts needs to be adjusted to accept "empty" / null? I gave a few options a try, but nothing would resolve the error message in `utils.ts`